frontend/app/src/types/websocket-messages.ts

```diff
@@ -47,14 +47,16 @@ export enum SocketMessageType {
   LEGACY = 'legacy',
   BALANCES_SNAPSHOT_ERROR = 'balance_snapshot_error',
   ETHEREUM_TRANSACTION_STATUS = 'ethereum_transaction_status',
-  PREMIUM_STATUS_UPDATE = 'premium_status_update'
+  PREMIUM_STATUS_UPDATE = 'premium_status_update',
+  LOGIN_STATUS = 'login_status'
 }
 
 interface MessageData {
   [SocketMessageType.LEGACY]: LegacyMessageData;
   [SocketMessageType.BALANCES_SNAPSHOT_ERROR]: BalanceSnapshotError;
   [SocketMessageType.ETHEREUM_TRANSACTION_STATUS]: EthereumTransactionQueryData;
   [SocketMessageType.PREMIUM_STATUS_UPDATE]: PremiumStatusUpdateData;
+  [SocketMessageType.LOGIN_STATUS]: z.ZodAny;
```

Comment:
Shouldn't the frontend PR be the one changing it? Also the tests failed but I don't know if is related. If you don't need this in this PR I would let the frontend PR handle this part

Reply:
Yes, will delete. I haven't removed this before since thought that Luki will put his changes in this pr